resources/tests/msr/wrmsr_list.txt

```diff
@@ -14,7 +14,6 @@
 0x26d 0x1
 0x26e 0x1
 0x26f 0x1
-0x277 0x7040600070407
```

Comment:
Can I ask why writing to MSR index 0x277 results in a vm crash?
And how did we choose the list of MSRs to be modified?

Reply:
AFAIK the wrmsr_list.txt was created by flipping 1 bit of the default value.
There could be MSRs which need to be update together and this could be the case here.
But to find a working set of MSRs to update we would have to go through reference doc which doesn't seem to be in the scope of the current task. Since the purpose of this file is to have some working combination of MSRs that we can update before running the snapshot test and also to have the file common for all Intel and AMD templates we decided to remove this 1 line.
As you see we don't update all the MSRs in this file for the same reason i.e. they are either reserved or need to be updated in a specific sequence.
Like in this case 0x277 is Page attribute table, its default value is 0x7040600070406 and as per AMD ref doc 3 is reserved value for bits 2:0.